pykickstart/options.py

```diff
@@ -149,6 +156,9 @@ def __init__(self, *args, **kwargs):
         if addVersion:
             kwargs['description'] = "\n.. versionadded:: %s\n\n%s" % (version,
                                                                     kwargs['description'])
+        if removed:
+            kwargs['description'] = "\n.. versionremoved:: %s\n\n%s" % (removed,
```

Comment:
This is not used in the example. Is there really a use case for it?

Reply:
Oops, we already have the `KSOptionParser.remove_argument()` function that handles doing that. I'll drop that part, thanks!